lib/jsanalyze.js

```diff
@@ -168,8 +168,10 @@ exports.analyzeJs = function analyzeJs(contents, opts = {}) {
 	}
 
 	let sourceFileName = opts.filename; // used to point to correct original source in sources/sourceRoot of final source map
+	let sourceMap = (sourceFileName && sourceFileName.includes('/node_modules/')) ? false : opts.sourceMap;
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be. -->
**suggestion (non-blocking):** Super nitpick‏

```suggestion
	const sourceMap = (sourceFileName && sourceFileName.includes('/node_modules/')) ? false : opts.sourceMap;
```